### PR TITLE
fix(chat): open chat or prompt bars on accepted sharing (Issue #1296)

### PR DESCRIPTION
**Pull request by likeamike**
**Description:**

Open chatbar or promptbar when sharing is accepted

Issues:

- Issue #1296 

**UI changes**

<Please, provide Screenshots or Figma links>

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1408.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/store/share/share.epics.ts [hunk 1]
```diff
@@ -270,6 +270,8 @@ const acceptInvitationEpic: AppEpic = (action$) =>
                 ShareActions.acceptShareInvitationSuccess({
                   acceptedId: ApiUtils.decodeApiUrl(acceptedIds[0].url),
                   isFolder: isFolderId(data.resources[0].url),
```
Comment (Alexander-Kezik) [inline apps/chat/src/store/share/share.epics.ts:272]: maybe we should check if folder is a chat/prompt folder?

**Review by Alexander-Kezik — DISMISSED** [review]

File: apps/chat/src/store/share/share.epics.ts (hunk 1 above)
Comment (likeamike) [reply apps/chat/src/store/share/share.epics.ts:272]: I'm not sure if I understand you correctly, but I believe that this three flags should work good enough. Also I pushed one more commit and reused these isPromt and isConversation near to isFolder further down the code.

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1408.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by Alexander-Kezik — APPROVED** [review]

**Comment by likeamike** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1408.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**